api/python/quilt3/workflows/config-1.schema.json

```diff
@@ -7,6 +7,25 @@
     "workflows"
   ],
   "additionalProperties": false,
+  "definitions": {
```

Comment:
do we have duplicate schema files in python code and catalog?

Reply:
yes